src/Configuration/ConfigurationInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCacheBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCacheBundle\Configuration;
+
+/**
+ * ConfigurationInterface.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+interface ConfigurationInterface
```

Comment:
looking at how we use this in Php8AttributesListener i would prefer to do a `match` on $instance to see if it is one of the 3 known attribute classes and if so add it with the corresponding alias name. this interface is not really adding value anymore i think.